aptos-move/aptos-gas/Cargo.toml

```diff
@@ -10,12 +10,5 @@ publish = false
 edition = "2021"
 
 [dependencies]
-move-binary-format = { git = "https://github.com/move-language/move", rev = "e97dc204943776e80a16255432ee4e92c4e08652" }
```

Comment:
This was discussed on the channel, for now it is not unnecessary, somehow related to feature unification.

Reply:
just paste the link to the discussion https://aptos-org.slack.com/archives/C036X27DZNG/p1658952159877049